src/server/admin/decommission-service/controllers/index.js

```diff
@@ -0,0 +1,17 @@
+import { decommissionFormController } from '~/src/server/admin/decommission-service/controllers/form.js'
```

Comment:
I'm trying to remove all the `index.js` files. They are adding unessacary complexity. You can just import all these in the routes file directly